packages/grafana-data/src/transformations/transformers/calculateField.test.ts

```diff
@@ -948,4 +948,53 @@ describe('calculateField transformer w/ timeseries', () => {
       expect(data.fields[1].values[3]).toBeCloseTo(0.6666666, 4);
     });
   });
+
```

Comment:
@samjewell -- this is likely the easiest way to test options:

![2023-12-04_21-57-30 (2)](https://github.com/grafana/grafana/assets/705951/ce7a7bd7-61ff-4d3d-9f26-ae08eaae66f7)

and/or just console.log() the results :)

Reply:
Thanks a lot, this was a great help.

Have you installed a Jest test-runner plugin @ryantxu ? I saw your video, then installed `Jest Runner`, but I get an error:
```
node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module '/Users/samjewell/fana/grafana/node_modules/jest/bin/jest.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.10.0
```

Any tips?